src/Client/Web/Bit.Client.Web.BlazorUI/Components/Rating/BitRating.razor.cs

```diff
@@ -58,9 +75,14 @@ protected override async Task OnInitializedAsync()
             await base.OnInitializedAsync();
         }
 
-        protected virtual void HandleClick(int index)
+        protected virtual async Task HandleClick(int index)
         {
             FillRating(index);
+
+            if (IsEnabled is false) return;
+            Value = index;
```

Comment:
this is not the correct way to change a 2-way bound parameter.
please check the implementations in the BitCheckBox component.

Reply:
again, it's not resolved